app/services/datadog_services/list_users.rb

```diff
@@ -0,0 +1,27 @@
+module DatadogServices
+  class ListUsers
+    # resource path
+    VAR_PATH = "/api/v2/users"
+    def initialize
+      @conn = Faraday.new(
+        url: "https://api.datadoghq.com",
+      )
+    end
+
+    def call(params = {})
+      response = @conn.get do |req|
```

Comment:
You can define attribute reader to avoid instance variable typo mistake
example
```
attr_reader :conn
```